datasets/gov/aaa/integruotas_monitoringas.csv

```diff
@@ -0,0 +1,39 @@
+id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri,title,description
+,datasets/gov/aaa/integruotas_monitoringas,,,,,,,,,,,,,
+,,,,,,,,,,,,,,
+,,,,Tyrimai,,,,,,2,,,Tyrimai,
+,,,,,tad_id,string,,,,4,open,,Tyrimo id,
+,,,,,tad_laikotarpis_nuo,date,D,,,4,open,,Tyrimo laikotarpis nuo,
+,,,,,tad_laikotarpis_iki,date,D,,,4,open,,Tyrimo laikotarpis iki,
+,,,,,tad_gis_taskas,"geometry(point, 3346)",,,,3,open,,Tyrimo vieta,
+,,,,,vkl_pavad,string,,,,3,open,,Veiklos pavadinimas,
+,,,,,ttr_pavad,string,,,,4,open,,Tyrimų terpės pavadinimas,
+,,,,,,enum,,,"""Aplinkos oras""",,,,Aplinkos oras,
+,,,,,,,,,"""Biota""",,,,Biota,
+,,,,,,,,,"""Paviršinis vanduo""",,,,Paviršinis vanduo,
+,,,,,,,,,,,,,,
+,,,,TyrimuSudedamosios,,,,,,2,,,Tyrimų sudedamosios,
+,,,,,tsd_tad_id,string,,,,4,open,,Tyrimo sudedamosios id,
+,,,,,tsd_tyrimo_tipas,string,,,,4,open,,Tyrimo sudedamosios tyrimo tipas,
+,,,,,,enum,,,"""CHE""",,,,Cheminiai tyrimai,
+,,,,,,,,,"""FIZ""",,,,Fizikiniai tyrimai,
+,,,,,tad_laikotarpis_nuo,date,D,,,4,open,,Tyrimo laikotarpis nuo,
+,,,,,tad_laikotarpis_iki,date,D,,,4,open,,Tyrimo laikotarpis iki,
+,,,,,tad_gis_taskas,"geometry(point, 3346)",,,,3,open,,Tyrimo vieta,
+,,,,,,,,,,,,,,
+,,,,TyrimuRezultatai,,,,,,4,,,Tyrimų rezultatai,
+,,,,,trz_id,string,,,,4,open,,"Tyrimų rezultato id",
+,,,,,trz_neapibr,number,,,,3,open,,"Parametras, apibrėžiantis tyrimo rezultato tikslumą",
```

Comment:
Ar šis parametras nereferuoja į `model=ParametraiVienetai`, nes 6iame modelyje kalba eina apie parametrus.

Reply:
Ne, tarp šių lentelių nėra bendrų ryšių